plugin-sftp/src/main/java/ca/on/oicr/gsi/shesmu/sftp/SshConnectionPool.java

```diff
@@ -111,12 +116,13 @@ public void configure(String host, int port, String user) {
 
   @Override
   public PooledSshConnection get() {
+    int attempts = 100;
```

Comment:
a hundred? That seems a lot

Reply:
:man_shrugging: It's less than infinity.